packages/dataviews/src/dataviews-layouts/table/style.scss

```diff
@@ -7,11 +7,6 @@
 	color: $gray-700;
 	margin-bottom: auto;
 
-	a {
-		text-decoration: none;
-		color: $gray-900;
-		font-weight: 500;
-	}
```

Comment:
The links within the dataviews fields shouldn't be that opinionated, they should just inherit the global link styles applied in the current app.

This is the change this is the most impactful in this PR but I didn't notice any regression.

Reply:
It seems you don't see a regression because we have similar opinionated styles for `dataviews-view-grid__primary-field`

Reply:
yes, I guess these were needed before the introduction of the primary fields.